stubs/pywin32/win32/win32api.pyi

```diff
@@ -91,7 +105,7 @@ def GetDomainName() -> str: ...
 def GetEnvironmentVariable(variable): ...
 def GetEnvironmentVariableW(Name) -> str: ...
 def GetFileAttributes(pathName: str): ...
-def GetFileVersionInfo(Filename: str, SubBlock: str) -> None: ...
+def GetFileVersionInfo(Filename: str, SubBlock: str) -> _FileVersionInfo: ...
```

Comment:
It looks like these parameters are actually positional-only, so let's fix that while we're here. We can't use PEP-570 syntax yet in typeshed as we still support Python 3.7, so this is the py37-compatible way of marking parameters as positional-only:

```suggestion
def GetFileVersionInfo(__Filename: str, __SubBlock: str) -> _FileVersionInfo: ...
```